tests/utils.py

```diff
@@ -564,3 +566,31 @@ def create_cirros_vm(
         if wait_running:
             running_vm(vm=vm, wait_for_interfaces=False)
         yield vm
+
+
+def verify_cpumanager_workers(schedulable_nodes):
+    """Verify cluster nodes have CPU Manager labels."""
+    LOGGER.info("Verifying cluster nodes have CPU Manager labels")
+    failures = []
+    if not any(node.labels.cpumanager == "true" for node in schedulable_nodes):
+        failures.append("Cluster does not have CPU Manager enabled on any node")
```

Comment:
here and in other functions - no need to create/append list `failures`, you can just return what need to inside the `if`